.editorconfig

```diff
@@ -0,0 +1,11 @@
+
+root = true
+
+[*]
+indent_style = true
+end_of_line = lf
```

Comment:
Okay. Have we discussed making the same thing for all plugins/lua code included in the build?